widgetssdk/src/main/java/com/glia/widgets/filepreview/ui/State.java

```diff
@@ -0,0 +1,240 @@
+package com.glia.widgets.filepreview.ui;
+
+import android.graphics.Bitmap;
+
+class State {
```

Comment:
Name can be improved

Reply:
its package locked, what do you mean by imported?

Reply:
https://docs.oracle.com/javase/tutorial/java/javaOO/accesscontrol.html